updated copy on list page, moved completed projects to proper section…

kungfuchicken opened the pull request:
…, added new projects.

@smallrussian and @lmamillapalli786 Please update the pages I created for your respective projects with:
- [ ] correct 
  - [ ] names of all developers
  - [ ] links to all developer GitHub profiles.
  - [ ] links to all developer LinkedIn profiles if possible/preferred
- [ ] updated descriptions as needed
- [ ] ensure "Technologies Used" section updated
- [ ] create an architecture diagram
- [ ] ensure your project has a valid license identified in the repo and on the about page.
- [ ] add a Get Involved section and any other relevant sections you find on other portfolio pages.

You can look at other portfolio pages for examples of what we're looking for.

lmamillapalli786 reviewed (commented):
I have added the changes regarding the project_open_project and waiting for @kungfuchicken to merge the changes to the main branch and also looking for feedback if anythings needs to be changed

kungfuchicken commented:
> I have added the changes regarding the project_open_project and waiting for @kungfuchicken to merge the changes to the main branch and also looking for feedback if anythings needs to be changed



> I have added the changes regarding the project_open_project and waiting for @kungfuchicken to merge the changes to the main branch and also looking for feedback if anythings needs to be changed

it’s getting there. There a few things that need to be fixed ASAP:
- [ ] The source code link has a trailing slash
- [ ] The LinkedIn links are not rendering as the LinkedIn icon image.
- [ ] You need to pick a license for the project. This needs to be set on the portfolio pages, but (more importantly) it needs to be added to the project repo itself. I will suggest the LGPL 3 licnese. https://opensource.org/license/lgpl-3-0
- [ ] The technical information should be formatted with  Markdown https://www.markdownguide.org/
